lib/improver/tests/utilities/__init__.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
```

Comment:
I am happy if they have nothing in them. They could be useful if they had a docstring to describe their package - otherwise probably not.